test/include/test/bench.h

```diff
@@ -23,7 +23,7 @@ fmt_nsecs(uint64_t usec, uint64_t iters, char *buf) {
 	uint64_t nsecs_per_iter1000 = nsec1000 / iters;
 	uint64_t intpart = nsecs_per_iter1000 / 1000;
 	uint64_t fracpart = nsecs_per_iter1000 % 1000;
-	malloc_snprintf(buf, FMT_NSECS_BUF_SIZE, "%"FMTu64".%03"FMTu64, intpart,
+	malloc_snprintf(buf, FMT_NSECS_BUF_SIZE, "%" FMTu64 ".%03" FMTu64, intpart,
```

Comment:
True, the spaces are not mandatory. Yet it will report a warning when compiling with c++11: `test/include/test/bench.h:55:16: warning: invalid suffix on literal; C++11 requires a space between literal and string macro [-Wliteral-suffix]
  malloc_printf("%"FMTu64" iterations, %s=%" FMTu64 "us (%s ns/iter), "` 
Thus, I modified it for better compatibility just in case.